lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -20,6 +20,8 @@ $package = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $
 $force = Get-AnsibleParam -obj $params -name "force" -type "bool" -default $false
 $version = Get-AnsibleParam -obj $params -name "version" -type "str"
 $source = Get-AnsibleParam -obj $params -name "source" -type "str"
+$source_username = Get-AnsibleParam -obj $params -name "source_username" -type "str" -failifempty ($source -ne $null)
```

Comment:
You should still be able to specify source without a username/password. Just take out the `-failifempty` section to allow that.

Reply:
@jborean93 
Does this not mean you can't provide a username and password without defining a source?
Source can still be on it's own, no?

Reply:
Yes you can define a source without a password, the current logic `-failifempty ($source -ne $null)` means that the module will fail if `source` is defined and the username is not defined and you want the opposite.